shared/common-adapters/custom-emoji.native.tsx

```diff
@@ -11,7 +11,10 @@ const CustomEmoji = (props: Props) => {
     width: size,
   }
   return (
-    <Box2 direction="vertical" style={Styles.collapseStyles([dimensions, styles.container])}>
+    <Box2
+      direction="vertical"
+      style={Styles.collapseStyles([dimensions, styles.container, !!props.addTopMargin && styles.topMargin])}
```

Comment:
@adamjspooner @hyperobject I had to add this `addTopMargin` prop since the margin pushes the custom emoji off in a few places. It seems service-decoration is the only place that should have this margin. Please let me know if I missed anything here!

Reply:
Works for me. 👍